### PR TITLE
Translate contribute/review/reviewing-prs into Japanese

### LINKED ISSUE #23947: Translate contribute/review/reviewing-prs into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [contribute/review/reviewing-prs](https://kubernetes.io/docs/contribute/review/reviewing-prs) into Japanese.

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by shuuji3**
This PR translates [Reviewing pull requests](https://kubernetes.io/docs/contribute/review/reviewing-prs/) into Japanese. 

Closes #23947. (Sorry for taking 5 months (!) to close the issue.)

🌐 Page preview: https://deploy-preview-26674--kubernetes-io-master-staging.netlify.app/ja/docs/contribute/review/reviewing-prs/

### CODE REVIEW

File: content/ja/docs/contribute/review/reviewing-prs.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+<!-- body -->
+
+## はじめる前に
+
+レビューを始める前に、以下のことを心に留めてください。
+
+- [CNCFの行動規範](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)を読み、いかなる時にも行動規範にしたがって行動するようにする。
+- 礼儀正しく、思いやりを持ち、助け合う気持ちを持つ。
+- 変更点だけでなく、PRのポジティブな側面についてもコメントする。
```
Comment (oomichi) [inline content/ja/docs/contribute/review/reviewing-prs.md:27]: By comparing https://github.com/kubernetes/website/blob/master/content/en/docs/contribute/review/reviewing-prs.md the following line seems missing:
```
Be empathetic and mindful of how your review may be received.
```

**Review by oomichi — COMMENTED** [review]
Thanks for doing this, most part seems good for me.
just one comment.

/cc @oomichi

File: content/ja/docs/contribute/review/reviewing-prs.md (hunk 1 above)
Comment (shuuji3) [reply content/ja/docs/contribute/review/reviewing-prs.md:27]: @oomichi Thank you for your careful review! It's my mistake. I updated the translation to include this sentence too.

**Review by oomichi — COMMENTED** [review]
Thanks for updating.
This is very nice translation for me.

/lgtm

**Comment by YuikoTakada** [comment]
Thank you for your PR! LGTM 👍

**Pull request merged**